evap/evaluation/templates/contact_modal.html

```diff
@@ -43,50 +48,9 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
         </div>
     </div>
 </div>
+<script type="module" src="/static/js/contact_modal.js"></script>
```

Comment:
I can remove this and my browser still does everything correctly. Is it necessary? If not, we should probably remove it.

(edit: we concluded that it can probably removed here here -- at least it still works in chrome and firefox then)

(also, /static/ here, where the `static` template tag is used below.)